src/javascript/static/pages/affiliate_ib_landing.js

```diff
@@ -1,8 +1,11 @@
 const Url         = require('../../../javascript/_common/url.js');
 const TabSelector = require('../../_common/tab_selector');
+const getLanguage = require('../../_common/language.js').get;
 
 const AffiliateIBLanding = (() => {
+    const current_language = getLanguage().toLowerCase();
```

Comment:
In binary Chinese and traditional Chinese having ` _ ` but in deriv.com we have `-` i.e `zh-cn` in deriv but `zh_cn` so we need to also replace that. 🙏 

```suggestion
    const current_language = getLanguage().toLowerCase().replace(/_/g, '-');
```

Reply:
Goot catch, thanks @mitra-deriv 🙏